getting_started/step_by_step/ui_main_menu.rst

```diff
@@ -12,7 +12,7 @@ scenes step-by-step using the engine's UI system:
 You will learn how to design game UI efficiently, and how to use Godot's
 Control nodes. This page focuses on the visual part: everything you do
 from the editor. To learn how to code a life bar,
-read :doc:`ui_game_user_interface`
+read :doc:`ui_code_a_life_bar`
```

Comment:
There's nothing about how to code in ui_game_user_interface tutorial but in ui_code_a_life_bar one.